colosseum/wrappers.py

```diff
@@ -0,0 +1,53 @@
+from collections import Sequence
+
+from .constants import INHERIT
+
+
+class BorderSpacing:
+    """
+    Border spacing wrapper.
+
+    Examples:
+        BorderSpacing('inherit')
```

Comment:
This seems like an odd case to capture *inside* BorderSpacing. Is there a reason to add INHERIT support to border spacing, rather than treating the value for border_spacing to be either INHERIT or a BorderSpacing object?

Reply:
Fixed!